lib/ansible/modules/cloud/amazon/ec2_asg_facts.py

```diff
@@ -265,6 +294,7 @@ def find_asgs(conn, module, name=None, tags=None):
                         "protected_from_scale_in": false
                     }
                 ],
+                "launch_config_name": "public-webapp-production-1",
                 "launch_configuration_name": "public-webapp-production-1",
```

Comment:
Yes, that's exactly it. Not sure it's needed tbh, there are so many other inconsistencies.